packages/charts/src/charts/heatmap/heatmap-chart.component.tsx

```diff
@@ -1,17 +1,21 @@
 import React, { FC, useRef } from 'react';
 import { motion, AnimatePresence } from 'framer-motion';
-import { Layout, Tooltip, ColorMode, RangeType } from '@keen.io/ui-core';
+import { Layout, Tooltip, ColorMode, RangeType, Text } from '@keen.io/ui-core';
 import { useTooltip } from '@keen.io/react-hooks';
-import { ScaleSettings, TooltipFormatter } from '@keen.io/charts-utils';
+import {
+  formatValue as valueFormatter,
+  getFromPath,
+  ScaleSettings,
+  TooltipFormatter,
+} from '@keen.io/charts-utils';
 
-import Heatmap from './heatmap.component';
-import TooltipContent from './tooltip-content.component';
-import ShadowFilter from './shadow-filter.component';
+import Heatmap from './components/heatmap/heatmap.component';
```

Comment:
Please, create index file with exports from `components` directory. Then in code:
```
import { Heatmap, ShadowFilter } from './components';
```